kythe/java/com/google/devtools/kythe/analyzers/java/JavaEntrySets.java

```diff
@@ -159,8 +159,12 @@ public VName getNode(
       if (config.getVerboseLogging()) {
         logger.atWarning().log("%s", msg);
       }
-      Diagnostic.Builder d = Diagnostic.newBuilder().setMessage(msg);
-      return emitDiagnostic(d.build()).getVName();
+      Diagnostic d = Diagnostic.newBuilder().setMessage(msg).build();
+      VName dVName =
+          getUseCompilationCorpusAsDefault()
+              ? VName.newBuilder().setCorpus(getCompilationVName().getCorpus()).build()
+              : null;
+      return emitDiagnostic(dVName, d).getVName();
```

Comment:
This is still calling the API that tags a file with a diagnostic.  I think this change needs to be addressed in KytheEntrySets.